tracetools_trace/tracetools_trace/tools/lttng_impl.py

```diff
@@ -427,6 +444,27 @@ def stop(
         raise RuntimeError(f"failed to stop tracing session '{session_name}': {error}")
 
 
+def record_snapshot(
+    *,
+    session_name: str,
+    **kwargs
```

Comment:
Very minor, but I usually use trailing commas for the last item in lists (list of parameters in this case)

```suggestion
    **kwargs,
```